### ISSUE TITLE
Add custom icon options.

### ISSUE THREAD

**Issue by mattmiklic** [wordpress-mobile/WordPress-Android#14476]
On iOS we offer custom icons as it's an option provided by the system. Speaking with @ParaskP7 we wanted to explore whether it's possible to bring this to Android as well. There are five color options, but I'll show two for the sake of brevity:

Pink:
![Pink](https://user-images.githubusercontent.com/1396552/115067948-96bf6580-9ebf-11eb-9cd2-3271f7622363.png)

Spectrum:
![Spectrum](https://user-images.githubusercontent.com/1396552/115067959-9a52ec80-9ebf-11eb-8994-967587b97430.png)

I've exported all five background colors as separate SVGs. The foreground SVG is included in this zip, but it's unchanged from what we're currently using.

[New-Icons.zip](https://github.com/wordpress-mobile/WordPress-Android/files/6327219/New-Icons.zip)

If we find out this is possible, then I'll look at how we should present the options in the UI.

**Comment by Dan-Q** [comment]
Even if Android doesn't natively support applications with multiple icon types, this might be somewhat achievable by adding a new Widget type (like the existing "just the icon" widget) but in other (or selectable!) color styles that can be pinned to the desktop. Basically-this approach is used by the FeedMe feed reader app to allow users to add a custom-colored shortcut icon to their desktop.

**Comment by eduardozulian** [comment]
If we implement this, it would be nice to also offer a [themed icon for Android 12](https://www.techrepublic.com/article/how-to-enable-themed-icons-in-android-12/)

**Comment by kuropixel** [comment]
Just wanted to add that some users at wpcom (5262120-zen) are also requesting the option to have custom icons or the ability to choose. 

I wanted to note that the Android DuckDuckGo app had this feature. Would it be difficult to implement?
![Screenshot_20220605-125723](https://user-images.githubusercontent.com/83020905/172034432-31141c91-2fd3-478f-b8a2-9f4954c8e735.png)
![Screenshot_20220605-125736](https://user-images.githubusercontent.com/83020905/172034435-906b7578-97ea-40db-a937-539400f6c012.png)

**Comment by mattmiklic** [comment]
I'd love for us to add custom icon choices for Android the way we do on iOS. I wasn't aware that it was possible on Android but I recently found out it's available in Pocket Casts for Android as well.

**Comment by joashrajin** [comment]
@mattmiklic Yes, with Pocket Casts you can choose your app icon. It generally works fine, but with the current implementation you end up with 2 icons - the original app icon and the custom icon that the user chose. 

We normally just recommend that users drag the custom icon to their home screen. Some Launchers also allow you to hide the duplicate app icon.

**Comment by kuropixel** [comment]
Another request. 5280194-zen

**Comment by joashrajin** [comment]
+1 at #5332325-zen

**Comment by thomashorta** [comment]
Just adding a bit of extra info here.

I just took a quick look at the behavior of both of them (DuckDuckGo and Pocket Casts) and they do it differently.

### DDG
DDG does seem to change the icon of their launcher activity so the launcher icon is replaced by the one you chose. I am not 100% sure how they achieve this, but it looks like good behavior, though the app kinda looks like it crashed when it changed the icon (probably a forced app restart, on purpose).

![image](https://user-images.githubusercontent.com/5091503/222444501-23bf71a8-c018-431a-b41a-dc315cbf7cde.png)

### Pocket Casts
Pocket Casts uses a different approach, [having several disabled activity aliases](https://github.com/Automattic/pocket-casts-android/blob/25760a68f3e3b7f77f8f261f5ff43758dd9c50a8/app/src/main/AndroidManifest.xml#L266-L433) for the launcher Activity, one for each icon, and enabling the one the user chooses, causing more than 1 launcher to appear, so you can choose the one you will put in your main screen. Having more than 1 launcher looks odd, even though the app does tell the user that multiple icons will appear in their launcher and that it might take a while for the icon to appear (probably depends on the launcher / device OEM).

![image](https://user-images.githubusercontent.com/5091503/222445010-c068bdbb-48ee-42b4-bf86-dfba75d18422.png)
